src/caretogether-pwa/src/GeneratedClient.ts

```diff
@@ -7425,6 +7445,11 @@ export abstract class ArrangementsCommand implements IArrangementsCommand {
             result.init(data);
             return result;
         }
+        if (data["discriminator"] === "DeletePlannedChildLocationChange") {

```

Comment:
This line (and similar ones above/below) look like they may benefit from the following changes:
- Switch the `if(data["discriminator" === ...)` lines to leverage a switch statement
- Instead of comparing to typed out strings, compare against a string enum (or another mechanism of your choice that would ensure typed-out strings are only ever set in a single place in the application, so as to reduce the potential for human error at a later point in development & potentially assist with intellisense/typing)

Reply:
See previous note on auto-codegen process for _GeneratedClient.ts_; otherwise, I 100% agree with you. 😊